microphone/drv_audio_pdm.c

```diff
@@ -161,8 +161,11 @@ ret_code_t drv_audio_init(nrf_pdm_mode_t mode)
 
 	pdm_cfg.mode        = local_mode;
 
-	// 20kHz
-	pdm_cfg.clock_freq	= 0x08800000;
+	//! Not officially supported value corresponding to PDM_CLK of 
+	//! NRF_PDM_FREQ_1280K, which will result in a 20KHz sample rate for HI
+	//! position. The HW in the uC is designed for this frequency but Nordic 
+	//! does not support it oficially as it's not considered tested enough 
```

Comment:
Should we add a link here also to the thread? So that we can find it easily again?